docs/README-short.txt

```diff
@@ -0,0 +1 @@
+An Ubuntu docker image containing the latest version of swift.
```

Comment:
Since this description is really meant to be more about the product itself, perhaps we could steal the following from the official swift website:

> Swift is a general-purpose programming language built using a modern approach to safety, performance, and software design patterns.

https://swift.org/about/